lib/form1010_ezr/service.rb

```diff
@@ -168,19 +169,26 @@ def add_financial_flag(parsed_form)
       end
     end
 
-    def log_validation_errors(parsed_form)
+    # @param [Hash] errors
+    def log_validation_errors(errors, parsed_form)
       StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.validation_error")
 
+      Rails.logger.error(
+        "10-10EZR form validation failed. Form does not match schema. Error list: #{errors}"
+      )
+
       PersonalInformationLog.create(
         data: parsed_form,
         error_class: 'Form1010Ezr ValidationError'
       )
     end
 
-    def log_and_raise_error(error, form)
-      log_submission_failure(form)
-      Rails.logger.error "10-10EZR form submission failed: #{error.message}"
-      raise error
+    def log_successful_submission(submission_id, veteran_initials)
+      Rails.logger.info(
+        '1010EZR successfully submitted',
+        submission_id:,
+        veteran_initials:
```

Comment:
Interesting. I haven't really seen veteran initials used anywhere. Does this help with debugging?

Reply:
It helps when we need to track submissions in VES. We can provide initials to the VES team so they can look for certain submissions without divulging PII.